test/sources/grib.jl

```diff
@@ -0,0 +1,219 @@
+using Rasters, Test, GRIBDatasets
+using Rasters: FileArray, FileStack, NCDfile, crs
+
+test_files = [
+    "era5-levels-members.grib", # OK
+    "fields_with_missing_values.grib", # OK
+    "forecast_monthly_ukmo.grib", # Failing because because there are multiple steps
```

Comment:
This dataset is more tricky to handle because it has multiple forecasts with different time steps, making it tricky to calculate the valid time. But that's actually a problem that must be solved in GRIBDatasets (I'll have a look at it). We can just ignore that for the moment. 

While using GRIBDatasets and Rasters for GRIB files, I also encountered other edge cases, but again I think this can be handled later on.

Reply:
Absolutely, making things mostly work and fixing as we go is mostly how Rasters.jl has developed. Working on 1% edge cases when 95% workflows aren't completed mediocre is not a good use of our time.